data/background/scene.inc

```diff
@@ -0,0 +1,20 @@
+;
+; first_nes (https://github.com/gregkrsak/first_nes/)
+; data/background/scene.inc
+;
+; Sample nametable to produce a background scene.
+; Uses data/tiles/smb1_chr.bin
+;
+; Please see first_nes/docs/README.md for license and attribution.
+
+
+_NAMETABLE:
+  .BYTE $24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24
+  .BYTE $24,$24,$24,$24,$24,$24,$24,$0f,$12,$1b,$1c,$1d,$24,$17,$0e,$1c ;first nes
```

Comment:
I can't recall-- I assume there is no underscore?

Also, maybe change the comment to clarify that it is actually referencing the glyphs for F-I-R-S-T and N-E-S, and not just making a vague comment/mention to the project name "first_nes".

Reply:
no underscore in the original that I could see.
perhaps a new tileset is in order?

and youre right, that documentation was poor at best.